### PR TITLE
Mc hr add avatars

### PR DESCRIPTION
**Pull request by freyjasdottir**
Adding avatars to the user model, enabling file upload via carrier wave, displaying in nav bar.

### CODE REVIEW

**Comment by freyjasdottir** [comment]
I'd like to merge this notwithstanding the hound violations, as they're all in auto generated files or in our hound_ignore file in master.

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -2,3 +2,14 @@
 *= require foundation_and_overrides
 *= require font-awesome
 */
+nav {
```
Comment (jraenorris) [inline app/assets/stylesheets/application.scss]: This should be in its own file. The `application.scss` file should just require other stylesheets, each of which should have its own file

File: .gitignore [hunk 2]
```diff
@@ -15,3 +15,4 @@
 .env
 /public/uploads
 .DS_Store
+.gitignore
```
Comment (jraenorris) [inline .gitignore]: you're git ignoring your .gitignore? I'd rethink that

File: app/assets/stylesheets/_avatar.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.avatar {
```
Comment (jraenorris) [inline app/assets/stylesheets/_avatar.scss]: This is in its own stylesheet and so nicely named - love it!

**Pull request merged**